capture-sdk/sdk/src/main/java/net/gini/android/capture/tracking/OnboardingScreenEvent.java

```diff
@@ -9,15 +9,15 @@
 /**
  * Events triggered on the onboarding screen.
  *
- * <p> All events trigger on both Screen and Component APIs.
+ * <p> All events trigger on Screen APIs.
  */
 public enum OnboardingScreenEvent {
     /**
-     * Triggers when the first onboarding page is shown.(<b>Screen API + Component API</b>)
+     * Triggers when the first onboarding page is shown.(<b>Screen API</b>)
```

Comment:
[Suggestion] Following the previous comments I think it makes sense to remove Screen API mentioning from small comments because it's now the only way of integration.